modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -684,19 +677,19 @@ GrainTracker::attemptGrainRenumber(MooseSharedPointer<FeatureData> grain, unsign
 
     // If we reach this part of the loop, there is no simple renumbering that can be done.
     mooseAssert(_unique_grains.find(target_it->_grain_id) != _unique_grains.end(), "Error in indexing target grain in attemptGrainRenumber");
-    MooseSharedPointer<FeatureData> target_grain = _unique_grains[target_it->_grain_id];
+    FeatureData & target_grain = *_unique_grains[target_it->_grain_id];
```

Comment:
Right, I'm still hesitantly moving to using `auto`. Scott Meyers makes a dang good case for it but it still freaks me out so I haven't put it everywhere just yet. This is probably a good candidate location.